modules/ding_ipe_filter/ding_ipe_filter.module

```diff
@@ -191,3 +205,33 @@ function _ding_ipe_filter_show_for_admin() {
 
   return $show_for_admin;
 }
+
+/**
+ * Get the current filter list.
+ *
+ * The panel panes selected in the administration UI and if not configured yet
+ * the default selections is used.
+ *
+ * @param bool $filter
+ *   If TRUE the field with value else all filters.
+ *
+ * @return array
+ *   The selected panes indexed by "category:subtype-name".
+ */
+function _ding_ipe_filter_selected_panes($filter = FALSE) {
+  $default = array(
+    'ding-:all_opening_hours' => 'ding-:all_opening_hours',
+    'ding-:popular' => 'ding-:popular',
+    'ding-:interaction_pane' => 'ding-:interaction_pane',
+    'ding-:serendipity_ting_object' => 'ding-:serendipity_ting_object',
+    'ding-:ding_tabroll-ding_frontpage_tabroll' => 'ding-:ding_tabroll-ding_frontpage_tabroll',
+    'event-panes:ding_event-ding_event_list_frontpage' => 'event-panes:ding_event-ding_event_list_frontpage',
+    'event-panes:ding_event-ding_event_simple_list' => 'event-panes:ding_event-ding_event_simple_list',
+    'groups-panes:ding_groups-ding_group_overview_simple' => 'groups-panes:ding_groups-ding_group_overview_simple',
+    'news-panes:ding_news-ding_news_frontpage_list' => 'news-panes:ding_news-ding_news_frontpage_list',
+    'ting:carousel' => 'ting:carousel',
```

Comment:
Is there any reason to have the same values as both key and value? It seems very verbose. I would expect a boolean value to be sufficient for our usecase e.g. 

```php
$default = array(
  'ding-:all_opening_hours' => TRUE,
  'ding-:popular' => TRUE,
// and so on

Reply:
It's the way drupal forms save the value in the database first time you save them... so...

Reply:
:-)